bundles/org.eclipse.ui/schema/perspectiveExtensions.exsd

```diff
@@ -308,6 +330,26 @@ If this attribute is not used, the view will be moveable, unless the perspective
       </complexType>
    </element>
 
+   <element name="editorOnboardingCommand">
```

Comment:
Shouldn't we also limit the number of commands. I think an onboarding text with e.g. 20 command is not longer so helpful.
I think we should stick to e.g. max. of 4 or 5 commands.

Reply:
went for 5

Reply:
> went for 5

Where do you process this in code?

Reply:
ModelPageLayout

https://github.com/eclipse-platform/eclipse.platform.ui/pull/656/files#diff-8ad222613d61876e357a83c18a1d930d92b0a5ccfe03d6c9c2686d26441eefd2R642

Reply:
Thanks. We should also document this limitation in the extension point docu.